File: Desktop/exchange/src/main/java/com/kjb04/exchange/api/model/Offer.java
package com.kjb04.exchange.api.model;

import com.google.gson.annotations.SerializedName;
import com.kjb04.exchange.DateParser;

import java.util.Date;

public class Offer {
    @SerializedName("usd_to_lbp")
    private Boolean usdToLbp;
    @SerializedName("amount_requested")
    private Float amountRequested;
    @SerializedName("amount_to_trade")
    private Float amountToTrade;
    @SerializedName("id")
    private Integer id;
    @SerializedName("username")
    private String username;
    @SerializedName("added_date")
    private String addedDate;

    public Boolean getUsdToLbp() {
        return usdToLbp;
    }
    public Float getAmountToTrade() {
        return amountToTrade;
    }
    public Float getAmountRequested() {
        return amountRequested;
    }
    public String getAddedDate() {
        return addedDate;
    }

    public String getUsername() { return username; }

    public Integer getId() {
        return id;
    }

    public Offer(Float amountRequested, Float amountToTrade, Boolean usdToLbp)
    {
        this.amountRequested = amountRequested;
        this.amountToTrade = amountToTrade;
        this.usdToLbp = usdToLbp;
    }
}

File: Android/exchange-android-EricNJ/app/src/main/java/com/example/currencyexchange/api/model/Offer.kt
package com.example.currencyexchange.api.model
import com.google.gson.annotations.SerializedName
class Offer {
    @SerializedName("id")
    var id:Int?=null
    @SerializedName("amount_requested")
    var amountrequested: Float? = null
    @SerializedName("amount_to_trade")
    var amounttotrade: Float? = null
    @SerializedName("usd_to_lbp")
    var usdToLbp: Boolean? = null
    @SerializedName("username")
    var user:String?=null
    @SerializedName("added_date")
    var addedDate:String?=null
}